src/api/helpers/get-application-by-reference-number/index.ts

```diff
@@ -0,0 +1,26 @@
+import { Application, Context } from '.keystone/types'; // eslint-disable-line
+
+/**
+ * getApplicationByReferenceNumber
+ * get ids of application sections by reference number
+ * returns application section ids or if not found, returns null
+ * @param {Number} referenceNumber
+ * @param {Context} context
+ * @returns {Application} application section ids or null
+ */
+const getApplicationByReferenceNumber = async (referenceNumber: number, context: Context): Application => {
```

Comment:
try-catch block?

Reply:
`console.info`?

Reply:
Not a blocker - `console.info` hasn't been added, but this is marked as resolved

Reply:
Got it in as

 ` try {
    console.info('Getting application by reference number');`

on my last commit

Reply:
Cannot see any `console.info.`